programmers/프린터.js

```diff
@@ -0,0 +1,17 @@
+function solution(priorities, location) {
+    const queue = priorities.map((x, index) => ({index : index , priority : x}));
+    let start = 0;
+    while (priorities.length > 0) {
+        const prior = Math.max(...priorities);
+        if (queue[0].priority !== prior) {
```

Comment:
여기도 마찬가지로 변수 선언 후 한 줄 띄워주세용
```suggestion
        const prior = Math.max(...priorities);
        
        if (queue[0].priority !== prior) {
```